### ISSUE TITLE
jQuery 3 support

### ISSUE THREAD

**Issue by JoshWillik** [fullcalendar/fullcalendar#3124]
The error message is `TypeError: rangeStart is null`.

**2.3.1**

[demo](http://jsbin.com/nisujeceyo/1/edit)

**2.3.2**

[demo](http://jsbin.com/laqesemewe/1/edit)

**Comment by JoshWillik** [comment]
The bug exists up to 2.6 as well

**Comment by arshaw** [comment]
You are using jQuery 3 beta.

I've update your fiddle to use latest FC and jQuery 2.2.4
http://jsbin.com/wifonupaba/1/edit?html,output
Works as expected.

Will revisit this when jQuery 3 comes out

**Comment by avindra** [comment]
@arshaw : I am also getting this on the jQuery 3.0 Release Candidate 1 (not beta).

**Comment by avindra** [comment]
Still encountering this in the final release of jQuery 3.0. I tried debugging a bit, and it seems to be related to the fact that `currentView.start` is null.

**Comment by ziemkowski** [comment]
This also prevents the calendar from loading for us.

Note that it happens even with [jQuery Migrate 3.0.0](http://jquery.com/download/#jquery-migrate-plugin), which is supposed to make jQuery backwards compatible.

**Comment by avindra** [comment]
@ziemkowski : I noticed that a hacky workaround is to click previous, then click next, which causes our events to render as expected. 

I may end up adding code to simulate the two clicks, because we are shipping soon and want to ship with jQuery 3.0.

**Comment by avindra** [comment]
This is the workaround I'll be using:

```js
// define this global somewhere that it would be in scope:

let stage = 0;
....
// add as a prop to the FC object

			viewRender(view, element) {
				// initial render is fixed now
				if(++stage == 3) return;
				const fc = element.parent().parent();
				const direction = stage == 1 ? 'prev' : 'next';
				fc.find(`.fc-${ direction }-button`).click();
			},
```


Unfortunately, the "today" button is broken using this hack. But good enough for us to ship.

**Comment by arshaw** [comment]
[Code complete](https://github.com/fullcalendar/fullcalendar/commits/v2.8.0-dev) and slated for v2.8.0 ([wip docs](https://github.com/fullcalendar/fullcalendar/commits/docs-2.0)). Will notify when released.

jquery 3 now works well

**Comment by arshaw** [comment]
Released in [v2.8.0](https://github.com/fullcalendar/fullcalendar/releases/tag/v2.8.0).

**Comment by avindra** [comment]
Thanks @arshaw, just tested and it works well :confetti_ball:
